versions.json

```diff
@@ -0,0 +1,21 @@
+{
+  "notice": "This file is not maintained outside of the main branch and should only be used for tooling.",
+  "versions": [
+    {
+      "version": "8.11.0",
```

Comment:
Why do we need the version here? This means we have to update this file on _every_ release to keep it up to date. If we just track branches, we only need touch this file when we feature freeze a new minor. We already have logic in `BwcVersions` to infer the versions of these branches based on known release versions so I'm not sure it's necessary to duplicate that here unless it has some other purpose.

Reply:
I removed version, too. I don't know of a use-case at the moment. We can consider adding it later if we need it.